sentrysql/operation.go

```diff
@@ -0,0 +1,22 @@
+package sentrysql
+
+import "strings"
+
+var knownDatabaseOperations = []string{"SELECT", "INSERT", "DELETE", "UPDATE"}
+
+func parseDatabaseOperation(query string) string {
+	// The operation is the first word of the query.
+	operation := query
+	if i := strings.Index(query, " "); i >= 0 {
+		operation = strings.ToUpper(query[:i])
+	}
+
+	// Only returns known words.
+	for _, knownOperation := range knownDatabaseOperations {
```

Comment:
You can make `knownDatabaseOperations` a map[string]bool (or map[string]struct{}), and skip the for range.

Something like:

```
	operation := strings.ToUpper(strings.Split(query, " ")[0])
	if _, exists := knownDatabaseOperations[operation]; exists {
		return operation
	}
	return ""
```

Reply:
Okay, might do this.